pokemonterminal/main.py

```diff
@@ -130,6 +133,9 @@ def main(argv):
         return
 
     if options.wallpaper:
+        if os.name == 'nt':
+            print("Setting wallpaper not supported on Windows yet.")
+            sys.exit(0)
```

Comment:
Instead of hardcoding a message for NT, I'd just leave our current thing do the job of returning an error when no adapter is available.

Reply:
The errors you get aren't great, and if you don't understand them it just looks like the program is program (because in a sense it is).
For example, omitting the traceback, for the slideshow you get: `[...] AttributeError: module 'os' has no attribute 'fork'` 
and for the wallpaper changing you get: 
```
'xprop' is not recognized as an internal or external command,
operable program or batch file.
[...]
subprocess.CalledProcessError: Command 'xprop -root -notype' returned non-zero exit status 1.
```
I think explicitly stating the feature is not supported is more user friendly and more clear

Reply:
I understand about slideshows and am fine about leaving it here, but the wallpaper error seems more like a bug.

Reply:
in `get_current_terminal_adapters()` when we check `is_compatible()` on each adapter,  `FehProvider.is_compatible()` runs `subprocess.check_output('xprop -root -notype', shell=True)` which throws the exception above. 
I don't know what the right solution is. We could try/catch it, but then you'd need the same kind of check in there to see did it raise because windows and the command doesn't exist, or because of some other actual problem